kuma/users/jinja2/users/includes/user_title_info.html

```diff
@@ -74,7 +74,7 @@ <h1 class="user-title">
     <li class="twitter"><i class="icon-twitter" aria-hidden="true"></i><a href="{{ detail_user.twitter_url }}" rel="me external" class="url">{{ _("Twitter") }}</a></li>
   {% endif %}
 
-  {% if accounts.github %}
+  {% if accounts.github and user.is_github_url_public %}
```

Comment:
``user`` is the request user. This should be ``detail_user.is_github_url_public``.

The second branch should also be updated, starting at line 82.

http://localhost:8000/en-US/profiles/Sheppy and http://localhost:8000/en-US/profiles/jwhitlock should have GitHub links, which should be not shown after the migration is applied.